cmd/tnf/generate/catalog/catalog.go

```diff
@@ -116,6 +121,81 @@ func emitTextFromFile(filename string) error {
 	return nil
 }
 
+// it turns a list of identifiers
```

Comment:
For autodoc purposes, we should replace "it" with the function name, in this case "createPrintableCatalogFromIdentifiers". I see a couple more instances of this pattern in the PR.

Reply:
you are right I have tried to improve the doc of the two main functions I created.